### PR TITLE
Update documentation for 1.4 release

### PR DESCRIPTION
**Pull request by emilevauge**
### Description

This PR update the documentation for the next 1.4 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,24 +60,30 @@ Run it and forget it!
...
-- Websocket support
-- HTTP/2 support
-- Retry request if network error
+- Websocket, HTTP/2, GRPC ready
+- Access Logs (JSON, CLF)
 - [Let's Encrypt](https://letsencrypt.org) support (Automatic HTTPS with renewal)
 - High Availability with cluster mode
 
+## Supported backends
+
```
Comment (timoreimann) [inline README.md]: No more BoltDB?

Comment (emilevauge) [reply README.md]: BoltDB is back!

Comment (timoreimann) [reply README.md]: Welcome back BoltDB! 🎉 

Do you also want to add it back to the top of the README and index files, respectively?

Comment (emilevauge) [reply README.md]: I simplified the top of the README using only the most used backends. So this is intended :)

Comment (timoreimann) [reply README.md]: Works for me!

**Pull request merged**